package/gem2rpm.yml

```diff
@@ -47,7 +45,7 @@
 #   install -D -m 0644 %{S:1} %{buildroot}%{_bindir}/gem2rpm-opensuse
 # ## used by gem2rpm
 :testsuite_command: |-
-   (pushd %{buildroot}%{gem_base}/gems/%{mod_full_name} && rake check:ci)
+   (pushd %{buildroot}%{gem_base}/gems/%{mod_full_name} && rake --verbose --trace check:ci)
```

Comment:
I originally took the `gem2rpm.yml` template and just replaced `rake test` by `rake check:ci`, the template is to blame :wink: